test/integration-test.sh

```diff
@@ -20,7 +20,10 @@ echo "Checking that the ecs-scripts installed correctly"
 configure-ecs-instance --help
 
 echo "Using gruntwork-install to install a module from the gruntwork-install repo and passing args to it via --module-param"
-gruntwork-install --module-name "dummy-module" --repo "https://github.com/gruntwork-io/gruntwork-installer" --tag "v0.0.17" --module-param "file-to-cat=$SCRIPT_DIR/integration-test.sh"
+gruntwork-install --module-name "dummy-module" --repo "https://github.com/gruntwork-io/gruntwork-installer" --tag "v0.0.25" --module-param "file-to-cat=$SCRIPT_DIR/integration-test.sh"
```

Comment:
This will fail initially as `v0.0.25` doesn't have `dummy-module`, right?

Reply:
Yes. I wasn't sure how you normally handle something like this, where you're testing against a tag that doesn't technically exist yet. I ran the tests locally to test.

Reply:
While testing, we switch `--tag` to the branch name we're testing with. Upon merge to `master`, the test may fail, so we switch to a release tag. 

Alternatively, perhaps we could get your new example checked into `master` with no test, and then in this PR, this test sets `--tag` to `master`?

Reply:
BTW, I'm realizing I highlighted the wrong line with this comment. `dummy-module` already exists in `gruntwork-instaler`, it's `args-test`, on the line below, that's the new one.